frontend/apps/reader/modules/readerkobolight.lua

```diff
@@ -0,0 +1,132 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
+local LeftContainer = require("ui/widget/container/leftcontainer")
+local GestureRange = require("ui/gesturerange")
+local Device = require("device")
+local Geom = require("ui/geometry")
+local Screen = require("device").screen
+local DEBUG = require("dbg")
+local UIManager = require("ui/uimanager")
+local Notification = require("ui/widget/notification")
+local T = require("ffi/util").template
+local _ = require("gettext")
+
+local DTAP_ZONE_KOBOLIGHTTOGGLE = {x = 0, y = 0.9375, w = 0.1, h = 0.0625 }
+local DTAP_ZONE_KOBOLIGHTSWIPE = {x = 0, y = 0.125, w = 0.1, h = 0.875 }
+
+local ReaderKoboLight = InputContainer:new{
+    steps = {0,1,1,1,1,2,2,2,3,4,5,6,7,8,9,10},
+    gestureScale = Screen:getHeight() * DTAP_ZONE_KOBOLIGHTSWIPE.h * 0.8,
+}
+
+function ReaderKoboLight:init()
+    self[1] = LeftContainer:new{
+        dimen = Geom:new{w = Screen:getWidth(), h = Screen:getHeight()},
+    }
+    self:resetLayout()
+end
+
+function ReaderKoboLight:resetLayout()
+    local new_screen_width = Screen:getWidth()
+    if new_screen_width == self._last_screen_width then return end
+    local new_screen_height = Screen:getHeight()
+    self._last_screen_width = new_screen_width
```

Comment:
Since the height changed here, do you want to update the `gestureScale` here too?